css/styles.css

```
@@ -258,10 +454,45 @@ width: 600px;
   text-transform: uppercase;
 }
 
+.about-us .about-us-services-list-title {
```

зачем зря увеличивать вес селектора, можно просто:
```suggestion
.about-us-services-list-title {
```